### DISCUSSION TITLE
Use with redwood in shared commonMain

### DISCUSSION THREAD

**Discussion by syxc** [icerockdev/moko-resources#671] [Q&A]
Without using Compose Multiplatform (iOS will increase the size of .ipa package by more than 20M when using it), is it possible to use String and other resources directly in the commonMain of the shared module, e.g. combining with redwood to realize UI sharing in commonMain (iOS, Android, etc.) will need to use String, Image and other resources? For example, if we combine [redwood](https://github.com/cashapp/redwood) to realize UI sharing in commonMain (iOS, Android, etc.), we will need to use String, Image and other resources.

**Comment by syxc** [comment]
Hey there, just a heads up that the iOS demo isn't quite compiling on Xcode 15.3.0 with my M1 Pro Mac. 
If it's not too much trouble, could you maybe tweak the example to sync up with the latest Redwood version, 0.10.0? 
I'd be super grateful for the assist. Cheers!

**Reply by Alex009** [reply]
hi. use https://github.com/icerockdev/moko-resources/releases/tag/release%2F0.24.0-beta-1 instead of 0.23.0

**Reply by syxc** [reply]
[redwood-sample](https://github.com/icerockdev/redwood-sample) can't compile and run successfully on Xcode 15.3.0.
